broker-plugin/plugin/src/main/sh/launch-broker.sh

```diff
@@ -2,6 +2,7 @@
 
 BROKER_DIR=${ARTEMIS_HOME}
 BROKER_CUSTOM_DIR=${BROKER_DIR}/custom
+# Deletes the last line so it wont start the broker? Necessary obscurity?
 sed -i '$ d' ${BROKER_DIR}/bin/launch.sh
```

Comment:
I'd prefer to remove the line from launch.sh rather than have the script delete it.  Is there some reason it needs to be this way?